src/ixdat/techniques/deconvolution.py

```diff
@@ -66,43 +64,40 @@ def extract_kernel(self, signal_name, cutoff_pot=0, tspan=None, t_bg=None):
         x_pot, y_pot = self.get_potential(tspan=tspan)
         x_sig, y_sig = self.get_signal(signal_name, tspan=tspan, t_bg=t_bg)
 
-        if signal_name == 'M32':
-            t0 = x_curr[np.argmax(y_pot > cutoff_pot)] #time of impulse
-        elif signal_name == 'M2':
+        if signal_name == "M32":
+            t0 = x_curr[np.argmax(y_pot > cutoff_pot)]  # time of impulse
+        elif signal_name == "M2":
             t0 = x_curr[np.argmax(y_pot < cutoff_pot)]
         else:
-            print('mass not found')
+            print("mass not found")
 
         x_sig = x_sig - t0
 
-        y_sig = y_sig[x_sig>0]
-        x_sig = x_sig[x_sig>0]
+        y_sig = y_sig[x_sig > 0]
+        x_sig = x_sig[x_sig > 0]
 
-        y_curr = y_curr[x_curr>t0]
-        x_curr = x_curr[x_curr>t0]
-        y_pot = y_pot[x_pot>t0]
-        x_pot = x_pot[x_pot>t0]
+        y_curr = y_curr[x_curr > t0]
+        x_curr = x_curr[x_curr > t0]
+        y_pot = y_pot[x_pot > t0]
+        x_pot = x_pot[x_pot > t0]
 
         kernel = Kernel(
-            MS_data = np.array([x_sig, y_sig]),
-            EC_data = np.array([x_curr, y_curr, x_pot, y_pot])
-            )
-
+            MS_data=np.array([x_sig, y_sig]),
+            EC_data=np.array([x_curr, y_curr, x_pot, y_pot]),
+        )
 
         return kernel
 
 
-
 class Kernel:
-    """
-    Kernel class implementing different functionalities for datatreatment of
-    kernel/impulse response data.
-    """
-    def __init__ (
+    """Kernel class implementing datatreatment of kernel/impulse response data."""
+
+    # TODO: Make class inherit from Measurement, add properties to store kernel
```

Comment:
This is the only TODO I can find about kernel. I'm worried we'll forget the other ideas if this is all.
TODO's work well as multi-line bits of class docstrings. (PyCharm highlights them yellow 😄 )